runtime/queries/metricsview_toplist.go

```diff
@@ -68,6 +68,10 @@ func (q *MetricsViewToplist) Resolve(ctx context.Context, rt *runtime.Runtime, i
 		return err
 	}
 
+	if mv.TimeDimension == "" {
+		return fmt.Errorf("metrics view '%s' does not have a time dimension", q.MetricsViewName)
+	}
+
```

Comment:
For toplist, it should only fail if `TimeStart` and/or `TimeEnd` are set – otherwise, it should succeed